cmake/projects/leveldb/hunter.cmake

```diff
@@ -15,8 +15,8 @@ hunter_add_version(
 hunter_add_version(
     PACKAGE_NAME leveldb
     VERSION "1.22"
-    URL "https://github.com/hunter-packages/leveldb/archive/v1.22-p0.tar.gz"
-    SHA1 "616e3816c0f44423136dcf286ae51417f0d75198")
+    URL "https://github.com/cpp-pm/leveldb/archive/refs/tags/v1.22-p1.tar.gz"
```

Comment:
why is the old version replaced? I think this should be a new entry versioned `1.22-p1` instead of `1.22`

Reply:
I have created a new entry now.
I made changes to the old entry earlier because it was broken, and it did not make sense to me to retain a broken version.